microsoftgraph/microsoft-graph-explorer-v4
# Task/persist sidebar selection

## Overview
Implements #1647 

### Demo

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

Check out this branch locally
Click on the history tab
Now click on the hamburger menu on the top left to hide the sidebar
Click on the hambugger menu again to restore the sidebar
Notice that the selection stays on the history tab
